automated/linux/ml-bench/pyarmnn-bench.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+
+usage() {
+    echo "Usage: $0 [-s <true|false>]" 1>&2
+    exit 1
+}
+
+while getopts "s:l:m:a:b:c:d:e:f:" o; do
+  case "$o" in
+    s) SKIP_INSTALL="${OPTARG}" ;;
+    a) LARMNN_TARBALL="${OPTARG}" ;;
+    d) TEST_DIR="${OPTARG}" ;;
+    *) usage ;;
+  esac
+done
+
+! check_root && error_msg "You need to be root to run this script."
+create_out_dir "${OUTPUT}"
+
+pkgs="ntp git wget curl libz-dev gcc g++ autoconf autogen libtool python3 python3-pip libjpeg-dev libtinfo5 libncurses5-dev libncursesw5-dev libboost-all-dev"
+dhclient
+install_deps "${pkgs}" "${SKIP_INSTALL}"
+
+if [ -n "${ARMNN_TARBALL}" ]; then
+    mkdir -p "${TEST_DIR}"
+    cd "${TEST_DIR}" || exit
+    wget -O armnn.tar.xz "${ARMNN_TARBALL}"
+    tar xf armnn.tar.xz
+    export BASEDIR="/home/buildslave/workspace/armnn-ci-build"
+fi
+
+if [ "${SKIP_INSTALL}" = false ]; then
+  cd "${BASEDIR}"/build || exit
+  n -s libprotobuf.so.15.0.0 ./libprotobuf.so.15
+  LD_LIBRARY_PATH="$(pwd)"
+  export LD_LIBRARY_PATH
+  pip3 install "${BASEDIR}"/build/dist/pyarmnn*.whl || exit
+  python3 "${BASEDIR}"build/scripts/download_test_resources.py
```

Comment:
you miss a '/' after '"${BASEDIR}"build/...' since the 'export BASEDIR="/home/..." doesn't include a trailing '/'.